clients/openpipeline/openpipeline.go

```diff
@@ -35,20 +34,6 @@ type ListResponse struct {
 	Editable bool   `json:"editable"`
```

Comment:
Just to clarify: we keep this such that users of the client could check easily if the config is editable without having to get each one?

Reply:
yes, maybe we are interested in that one